images/gh_runner/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Cache Holodeck image
+FROM ghcr.io/nvidia/holodeck:v0.1.1 AS holodeck
```

Comment:
Can we use build args to provide the versions instead of hardcoding here?

Reply:
The idea is to rely on dependabot for version management